.github/workflows/ci.yml

```diff
@@ -650,6 +654,7 @@ jobs:
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           $RETRY $MAVEN clean install ${MAVEN_FAST_INSTALL} -pl '!:trino-docs,!:trino-server-rpm'
+          $MAVEN dependency:get -Dartifact=org.testng:testng:5.10:pom -Dtransitive=false
```

Comment:
It's required by Surefire's TestNG extension, but isn't on the dependencies list since it's not actually a Trino dependency (not even transitive). I see `dependencies:go-offline' should get it as well.